_rules/SVG-image-has-name.md

```diff
@@ -0,0 +1,201 @@
+---
+name: SVG image has accessible name
+description: |
+  This rule checks that each SVG image element that is included in the accessibility tree, has an accessible name.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any element in the [SVG](https://www.w3.org/2000/svg) namespace with a [semantic role](#semantic-role) of either `img` `graphics-document` `graphics-object` `graphics-symbol`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element has an accessible name.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
```

Comment:
Verify accessibility support note.

Reply:
Just a note that the link https://www.w3.org/2000/svg goes to a short page recommending a redirect to the latest version here: http://www.w3.org/TR/SVG.  Suggest using this link instead.